framework/src/Volo.Abp.Security/Volo/Abp/Security/Claims/AbpClaimTypes.cs

```diff
@@ -44,15 +59,14 @@ public static class AbpClaimTypes
         public static string PhoneNumberVerified { get; set; } = "phone_number_verified";
 
         /// <summary>
-        /// Default: "tenantid".
+        /// Default: "tenant_id".
         /// </summary>
-        public static string TenantId { get; set; } = "tenantid";
-
+        public static string TenantId { get; set; } = "tenant_id";
 
         /// <summary>
-        /// Default: "editionid".
+        /// Default: "edition_id".
         /// </summary>
-        public static string EditionId { get; set; } = "editionid";
+        public static string EditionId { get; set; } = "edition_id";
```

Comment:
Do not change existing claim names. It is an unnecessary breaking change. It may even effect the existing access tokens obtained by clients, when we upgrade the ABP Framework